src/platforms/react-native/manual-setup/native-init.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Native Initialization
+sidebar_order: 4
+description: "Learn how to manually initialize the native SDKs."
+---
+
+By default, the React Native SDK will initialize the native SDK underneath when the `init` method is called on the JS layer. This currently has the limitation of not capturing native crashes that occur before the `init` method is called on the JS layer. You could also look to initialize the native SDKs yourself to be able to provide custom options above what the React Native SDK currently provides.
+
+To do this, set [autoInitializeNativeSdk](/platforms/react-native/configuration/options/#autoInitializeNativeSdk) to `false` in the init options:
+
+```javascript
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  autoInitializeNativeSdk: false,
+});
+```
+
+This will prevent the React Native SDK from initializing the native SDK automatically.
+
+<Alert level="info" title="Note">
```

Comment:
When using "Alert", the box is red and eye catching, so it shouldn't be labeled "Note". If it's just a Note, use the Note/Note construction. If you want to keep this as a red, eye-catching box, change the title to "Disabling the Native Layer" or something more useful that helps the reader understand why we're calling this out so prominently.